components/AddFormDialog/AddFormDialog.tsx

```diff
@@ -65,21 +68,35 @@ const AddFormDialog = ({
           inPreview: !f.isViewableByAdults && !f.isViewableByChildrens,
         }))
         .concat(
-          gForms.map((f) => ({
-            label: f.text,
-            href: `${f.value}${populateChildForm(
-              person.firstName,
-              person.lastName,
-              person.id,
-              user.email,
-              f.value
-            )}`,
-            system: false,
-            groupRecordable: false,
-            approvable: false,
-            inPreview: false,
-          }))
+          gForms
+            .map((f) => ({
+              label: f.text,
+              href: `${f.value}${populateChildForm(
+                person.firstName,
+                person.lastName,
+                person.id,
+                user.email,
+                f.value
+              )}`,
+              system: false,
+              groupRecordable: false,
+              approvable: false,
+              inPreview: false,
+            }))
+            .concat(
+              isFeatureActive('workflows-pilot')
```

Comment:
Update this so only users in the Google group can see this.

Reply:
Updated so will only display if feature flag is on and user is in workflow pilot.